src/hypervisor.adoc

```diff
@@ -288,7 +288,7 @@ HS-mode and U-mode.
 
 Field GVA (Guest Virtual Address) is written by the implementation
 whenever a trap is taken into HS-mode. For any trap (breakpoint, address
-misaligned, access fault, page fault, or guest-page fault) that writes a
+misaligned, access-fault, page-fault, or guest-page-fault) that writes a
```

Comment:
I think this change should be backed out, since these are nouns, not adjectives.